x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance.go

```diff
@@ -54,14 +58,34 @@ func WithdrawalRewardBalanceCallback(k Keeper, ctx sdk.Context, args []byte, que
 	k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_WithdrawalRewardBalance,
 		"Query response - Withdrawal Reward Balance: %v %s", withdrawalRewardBalanceAmount, tradeRoute.RewardDenomOnHostZone))
 
-	// Using ICA commands on the withdrawal address, transfer the found reward tokens from the host zone to the trade zone
-	if err := k.TransferRewardTokensHostToTrade(ctx, withdrawalRewardBalanceAmount, tradeRoute); err != nil {
+	// Split the withdrawal amount into a rebate, stride fee, and reinvest portion
+	feeInfo, err := k.CheckForCommunityPoolRebate(ctx, chainId, withdrawalRewardBalanceAmount)
+	if err != nil {
+		return errorsmod.Wrapf(err, "unable to check for community pool rebate")
+	}
+
+	// If there's a rebate portion, fund the community pool with that amount
+	if feeInfo.RebateAmount.GT(sdkmath.ZeroInt()) {
+		rebateToken := sdk.NewCoin(tradeRouteCallback.RewardDenom, feeInfo.RebateAmount)
+		if err := k.FundCommunityPool(ctx, hostZone, rebateToken, types.ICAAccountType_WITHDRAWAL); err != nil {
+			return errorsmod.Wrapf(err, "unable to submit fund community pool ICA")
+		}
+
+		k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_WithdrawalRewardBalance,
+			"Sending rebate tokens %v %s to community pool",
+			feeInfo.RebateAmount, tradeRoute.RewardDenomOnHostZone))
```

Comment:
tradeRouteCallback.RewardDenom is actually tradeRoute.RewardDenomOnRewardZone not on host zone which would be an ibc hash in this log statement

Reply:
are you suggesting to use `RewardDenomOnRewardZone` so that the logs don't have an ibc hash? If so I agree with you, but just double checking cause `RewardDenomOnHostZone` is the actual denom of the transfer

Reply:
Yes I was suggesting it for readability but also the variable from the callback tradeRouteCallback.RewardDenom what goes into that it the `RewardDenomOnRewardZone` as well